server/aws-lsp-codewhisperer/src/language-server/netTransformServer.ts

```diff
@@ -18,7 +25,48 @@ export const NetTransformServerFactory: (
             params: ExecuteCommandParams,
             _token: CancellationToken
         ): Promise<any> => {
-            //Placeholder for logic
+            try {
+                const client = createService(credentialsProvider)
+                const transformHandler = new TransformHandler(client, workspace)
+                switch (params.command) {
+                    case 'aws/qNetTransform/startTransform': {
+                        const userInputrequest = params as QNetStartTransformRequest
+                        logging.log('prepare artifact for solution: ' + userInputrequest.SolutionRootPath)
+                        return await transformHandler.startTransformation(userInputrequest)
+                    }
+                    case 'aws/qNetTransform/getTransform': {
```

Comment:
What is the purpose of `getTransform`? Is there a different response from `start` that are not the results?

Reply:
So the getTransformation is the polling api. This will give the status of transformation as Inprogess, Failed, etc . Based on that we show different progress on the client side.